addons/arsenal/functions/fnc_onArsenalClose.sqf

```diff
@@ -62,8 +54,23 @@ if (is3DEN) then {
         case 1: {GVAR(center) selectWeapon secondaryWeapon GVAR(center);};
         case 2: {GVAR(center) selectWeapon handgunWeapon GVAR(center);};
     };
+
+    if (!(isnull curatorCamera) && {ACE_player == player}) then {
+        curatorcamera cameraEffect ["internal","back"];
+    } else {
+        GVAR(camera) cameraEffect ["terminate","back"];
+
+        if (ACE_player == player) then {
+            player switchCamera GVAR(cameraView);
+        } else {
+            ACE_player switchCamera GVAR(cameraView);
+        };
```

Comment:
Uh...... if ACE_player==player then ACE_player equals player... 
`ACE_player switchCamera GVAR(cameraView);` ? 
Confused.

Reply:
Well, I have a perfect explanation for this....... _rewrites history_